Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -155,10 +156,10 @@ class JitBaseBlockCache
   virtual void WriteLinkBlock(const JitBlock::LinkData& source, const JitBlock* dest) = 0;
   virtual void WriteDestroyBlock(const JitBlock& block);
 
-  void LinkBlockExits(int i);
-  void LinkBlock(int i);
-  void UnlinkBlock(int i);
-  void DestroyBlock(int block_num, bool invalidate);
+  void LinkBlockExits(JitBlock& b);
```

Comment:
This should use something like `block`, not `b`. Ditto for `FinalizeBlock` and in the cpp function definitions as well.